packages/remediations/src/RemediationWizard.js

```diff
@@ -312,9 +312,9 @@ class RemediationWizard extends Component {
 
         return (
             <Wizard
-                isLarge = { true }
+                width="initial"
                 isValidated={ this.state.valid && (this.state.nameValid || !this.state.isNewSwitch) }
-                title="Remediate with Ansible"
+                title="Plan remediation"
```

Comment:
The mocks modify terminology but that is not in effect currently. The old terminology should be used as described in https://marvelapp.com/j3b9b7e/screen/63083483
I.e., this title should not change. Sorry for confusion.